### ISSUE TITLE
docker-compose update from 14 to 15 failing

### ISSUE THREAD

**Issue by rvalle** [nextcloud/docker#591]
Hi!
I installed NC 14 with docker-compose following the documentation.
I pulled NC 15 and the installation is broken.

I noticed that the volume I created includes also the source code of NC, and that a script is trying to upgrade NC code in my data volume.

As a docker user I expected that the data volumes would include data only. and that switching the image would magically work, or at least allow to go back to the previous image with NC14.

**Comment by J0WI** [comment]
Please provide some more details about the error message and your docker-compose setup.

**Comment by rvalle** [comment]
Sorry @J0WI I could not upgrade NC 14 to 15 with docker compose, and I finally desisted in favor of syncthing, despite of the tremendous potential of nextcloud.
I installed NC with docker compose following the provided documentation.
I just think that the docker upgrade process is not been tested.
As a route cause of the problem I think that nexcloud is recommending to make a data volume that includes the php application itself, therefore the upgrade depends on the successful execution  of an upgrade script. I think this breaks the docker microservice pattern, and brings this kind of problems.
